go/interpreter/lfvm/instructions_test.go

```diff
@@ -792,3 +793,85 @@ func TestTransientStorageOperations(t *testing.T) {
 		})
 	}
 }
+
+func TestExpansionCostOverflow(t *testing.T) {
+	memTestValues := []uint64{
+		maxMemoryExpansionSize,
+		maxMemoryExpansionSize + 1,
+		math.MaxUint64,
+	}
+
+	tests := map[string]struct {
+		op         func(*context)
+		stackSize  int
+		memIndexes []int
+		revision   tosca.Revision
```

Comment:
nit: revision is always the same value; no need to define it in the test case spec;